qt-models/diveplannermodel.cpp

```diff
@@ -1039,6 +1039,7 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 		copy_dive(&displayed_dive, current_dive);
 	}
 	mark_divelist_changed(true);
+	sort_table(&dive_table);
```

Comment:
hmm, so here we have a table sort. is this happening *after* the `init_decompression()` and `calculate_cns_dive()` calls?

Reply:
Yes, in the planner case this is happening after. It is happening at the moment when the user finally decides to save the planned dive to the dive list.